test/missing_flag_name.cc

```diff
@@ -23,3 +23,25 @@ TEST(FlagsTest, MissingFlagHelp) {
       "Missing flag 'help' for "
       "field 'test.MissingFlagHelp.s'");
 }
+
+TEST(FlagsTest, MissingSubcommandFlagName) {
```

Comment:
```suggestion
TEST(FlagsTest, MissingSubcommandName) {
```

Please make similar changes as necessary to other tests, thanks!